### PR TITLE
Update max fulfillment size

### PR DESCRIPTION
**Pull request by Siegrift**
https://api3dao.atlassian.net/browse/AN-784

### CODE REVIEW

**Comment by dcroote** [comment]
I saw the tests had failed so I updated them as my understanding was this is holding back `0.7`

File: packages/airnode-adapter/src/response-processing/extraction.test.ts [hunk 1]
```diff
@@ -295,14 +295,14 @@ describe('respects maximum encoded size', () => {
   });
 
   describe('encoding int256 arrays', () => {
-    const maxInt256ArrayLength = 30;
+    const maxInt256ArrayLength = 510;
 
     it('is under the limit', () => {
```
Comment (amarthadan) [inline packages/airnode-adapter/src/response-processing/extraction.test.ts:300]: Test case names are switched. It should say `exceeds the limit` and `exceeds the limit` one should say `is under the limit`.

Comment (amarthadan) [reply packages/airnode-adapter/src/response-processing/extraction.test.ts:300]: Same goes for the string tests above.

File: packages/airnode-adapter/src/response-processing/extraction.test.ts [hunk 2]
```diff
@@ -295,14 +295,14 @@ describe('respects maximum encoded size', () => {
   });
 
   describe('encoding int256 arrays', () => {
-    const maxInt256ArrayLength = 30;
+    const maxInt256ArrayLength = 510;
```
Comment (amarthadan) [inline packages/airnode-adapter/src/response-processing/extraction.test.ts:298]: I presume this is correct but out of curiosity, can somebody explain the math behind it? :smile:

**Review by amarthadan — APPROVED** [review]

File: packages/airnode-adapter/src/response-processing/extraction.test.ts (hunk 2 above)
Comment (dcroote) [reply packages/airnode-adapter/src/response-processing/extraction.test.ts:298]: Previously: `32 * 30 = 1024 - 64`
Now: `32 * 510 = 16384 - 64`

And my understanding for the reason to subtract 64 is:

```ts
const arrLen = ethers.utils.arrayify(ethers.utils.defaultAbiCoder.encode(['int256[]'], [[]])).byteLength;
console.log(arrLen);
```
Prints
```sh
64
```
Same as does:
```ts
const arrLen = ethers.utils.arrayify(ethers.utils.defaultAbiCoder.encode(['string'], [''])).byteLength;
```
hence:
https://github.com/api3dao/airnode/blob/5dd3b4a7dd387c413bee154c73f33f40511d14a8/packages/airnode-adapter/src/response-processing/extraction.test.ts#L278

**Pull request merged**